### ISSUE TITLE
M> Translate video Exclusive chat Lawrence Lerner & Nash Foster 

### ISSUE THREAD

**Issue by lapin7** [rchain/bounties#251]
# goal: Translate video [Exclusive chat Lawrence Lerner & Nash Foster with BlockchainBrad](https://www.youtube.com/watch?v=PwuV7ryhaXw)

The file name is: [[TRANSCRIPT][.srt][English] "Rchain Exclusive chat Lawrence Lerner & Nash Foster"](https://docs.google.com/document/d/1oTZVkYEhhnZtyU4DmNiiT3N39R0IVpHI92vP8kaltFs/edit)
It's in Folder: [/RChain/Members/Marketing-Community Dev/YouTube](https://drive.google.com/drive/folders/1zBZXvPMIC9yBneYI2c4x-xGwr-D1X1Ii)

- [x] create a complete transcript in English with timed links to subjects before 2018-01-28
- [x] translate transcript within 2 weeks in : 
- [x] Mandarin #264, 
- [x] French, #272 
- [x] Greek, #265
- [x] German #266, 
- [x] Spanish #263, 
- [ ] Portugese #282 
- [x] Dutch, #283 
- [ ] Serbo-Croat, #262 
- [x] Italian, #392 
- [x] Russian, #285 
- [ ] (or add other languages)
- [x] contact BlockchainBrad in order to add subtitles to his video 2018-01-29
- [ ] create "RHOC receivable promotor links"
- [ ] promote video on all channels and other outlets
- [ ] add link to your email-footer
- [ ] announce issue in Discord
- [ ] what else....

# budget: $10000 
# time: 2 weeks

**Comment by lapin7** [comment]
@truffard Le deadline est à définir avec les autres collaborateurs. La traduction doit être fait le plus vite possible. Mais d'abord il faire le transcript de toute la video en Anglais. J'espère que ce transcript en Anglais sera fini après le weekend.

**Comment by truffard** [comment]
Blockchain brad on le contacte sur github ou youtube?

**Comment by lapin7** [comment]
First we need a written transcript of all the spoken text on the video.
Maybe @entropee can help with that.
@truffard please try to get the contact details of BlockchainBrad from @llerner

**Comment by truffard** [comment]
@llerner ...ok im going away for 4 days but i will buckle down to it upon my return

**Comment by niwaa** [comment]
@lapin7 @truffard  If you need any help with french translations, let me know, I am a french native and familiar with technical terms (I am a dev). I am also pretty available these upcoming days.

**Comment by lapin7** [comment]
@truffard that's great. Could you also provide an English transcript in the first place and then a French one?

**Comment by zsluedem** [comment]
I can do with the chinese translations.

**Comment by lapin7** [comment]
@jimscarver @entropee can you help with your video club of Mattermost, Divvy, Social Ledger?

**Comment by niwaa** [comment]
@lapin7 Cool i'll sync-up with @truffard to see if he needs help first so there is no work overlap. Is there a process to follow to for task assignment/validation?  Or is everything done through commenting here and chatting in discord?

**Comment by lapin7** [comment]
@niwaa work between collaborators is best done through comments here (at least a summary of decisions between collaborators) But you can choose of course your own channels to communicate between each other.

@niwaa Edit first the transcript in a Google Doc for easy group editing. When the text is finished, put it into an .md for easy maintenance. I don't know the format for subtitling a Youtub video. But of course you can see for yourself what works best.

**Comment by lapin7** [comment]
@zsluedem please wait for the written English transcript or help to create the English transcript.

**Comment by zsluedem** [comment]
@lapin7 ok!
@niwaa @entropee @lapin7 I can help with the English transcript~Anybody create the transcript in the Google Doc already?

**Comment by niwaa** [comment]
@zsluedem I am almost done with the english transcript, I'll share the doc soon today for review.

**Comment by zsluedem** [comment]
@niwaa good

**Comment by lapin7** [comment]
Wonderful.
Now, all please go ahead with the translations to several languages. Please post here your start and progress, in order to avoid double work.
And don't forget to fill out [your reward % and the percentages](https://docs.google.com/spreadsheets/d/1uxuxx8YN17KCIWcH1cUoGuSm2hAnIya2iAc6wxoaq1o/edit#gid=950224463&range=C2045) of your fellow collaborators.

**Awesome**.

**Comment by Ojimadu** [comment]
I will get translations done in German and spanish. @niwaa You are doing the french translation right?

**Comment by zsluedem** [comment]
OK~!I am going to do the Chinese part~!

**Comment by Whisker17** [comment]
I think it is a big project.maybe we can cooperate with it,I have done some @zsluedem

**Comment by entropee** [comment]
Wow! I am too slow. I was about half way done with an english transcript. Great to see such excitement around the project. This really is a good video. We ought to try to organize a bit to be a sort of subtitle and translation response team. I'll start watching github closer. Anyway, while working on the project I found a command line program that will download and upload videos and their captions file. I tried it on a divvydao video and it worked!

**Comment by entropee** [comment]
For what it is worth, here is what I have done. I have corrected down to about 16 minutes and added the speakers initials in the left column.
https://docs.google.com/document/d/1YdDxltQAtdsuMJx9ePCFPS4jZRErha377DDWZuYneII/edit

**Comment by entropee** [comment]
Here is the command line program for dealing with youtube videos and subtitles. The .exe file for windows ran the first time. https://github.com/rg3/youtube-dl

**Comment by Tonyprisca13** [comment]
i will get translation done in mandarin

**Comment by Ojimadu** [comment]
@entropee The link you sent requires access permission.

**Comment by entropee** [comment]
Okay, let me try and fix the permission.

**Comment by entropee** [comment]
Try the link, now.

**Comment by Ojimadu** [comment]
Works now

**Comment by Ojimadu** [comment]
I think we should work on editin the document sent by @niwaa . There are still some error made by the text to speech converter that we need to correct before we can start working on the document.

**Comment by zsluedem** [comment]
@niwaa there are some mistakes I found in the doc.I have corrected them.

**Comment by zsluedem** [comment]
@Whisker17 and I are working together with the Chinese translation

**Comment by LiliNL** [comment]
cut this video in partitions

**Comment by kaka56** [comment]
@Ojimadu  I think  we should  work together  to translate  in German and Spanish  So it could be done on time

**Comment by Tonyprisca13** [comment]
I twittered brad in regard to subtitle on his video.

**Comment by niwaa** [comment]
@entropee Sorry didn't know you started something. Maybe we can merge the two? There are probably some remaining mistakes in the doc.

@Ojimadu Yes I can do the French translation today. You could review it if you have time.

**Comment by Viraculous** [comment]
I could get the summary/ briefings of the video because of its length, it many not be convenient for all but summary could be used as a preamble to the video in the channels. I can also work out its French translation.

**Comment by entropee** [comment]
@niwaa, no problem. I am glad to see these translations happening. I think it is very important that information about the project be widely available. Thanks to everyone that is helping with this!

**Comment by Mervyn853** [comment]
@entropee @lapin7 I helped set up the interview with Blockchain Brad with Nash and Lawrence, do let me know if you guys are done with transcribing / translations. I am in direct contact with Brad and I can get him to add the subtitles once its done

**Comment by lapin7** [comment]
Please don't start translations yet. Only after [this doc](https://docs.google.com/document/d/1oTZVkYEhhnZtyU4DmNiiT3N39R0IVpHI92vP8kaltFs/edit) is **FINAL**

The status and so are edited in this doc.

**Comment by lapin7** [comment]
@Mervyn853 please inform Blockchain Brad about our work and ask him if the final .srt file can be integrated with **his youtube video**

**Comment by Mervyn853** [comment]
@lapin7 Done, email has been sent. CC-ed you in HJ

**Comment by zsluedem** [comment]
@lapin7 @niwaa @entropee @Viraculous @kaka56 @Whisker17 
I have added some comment in the English transcript where I found it may be wrong. 
I think we should work together in this way. And review each others' opinions about the English transcript~!

**Comment by Tonyprisca13** [comment]
Here was the reply I got from BlockchainBrad on twitter "what language? you can download it, add them, and re-upload if you want."

**Comment by lapin7** [comment]
@Tonyprisca13 so how does "what language? you can download it" go?

**Comment by Mervyn853** [comment]
@lapin7 Check your email, Brad replied "Sure you can anything you need you go for it! No probs! Happy to help! " I suppose we are good to translate once its final.

**Comment by Tonyprisca13** [comment]
@lapin7 I think he meant that we can download the video, add the subtitles and re-upload them on YouTube.

**Comment by lapin7** [comment]
Great to hear that. I was afraid that our effort was for nothing. And I couldn't believe that translation was just a matter of some clicks.

**Comment by niwaa** [comment]
@zsluedem agreeing with all your corrections so far, and replied to comments. 

@Mervyn853 we will save the google doc as an uploadable .srt file on this repository when it is fully validated.

**Comment by lapin7** [comment]
Let's say that the transcript is approved, when there are no more edits done within the next 12 hours from now. We do some time boxing, because the translators have to get it done as well ASAP. The translators can prepare them selves already. 

The file name is: [[TRANSCRIPT][.srt][English] "Rchain Exclusive chat Lawrence Lerner & Nash Foster"](https://docs.google.com/document/d/1oTZVkYEhhnZtyU4DmNiiT3N39R0IVpHI92vP8kaltFs/edit)
It's in Folder: [/RChain/Members/Marketing-Community Dev/YouTube](https://drive.google.com/drive/folders/1zBZXvPMIC9yBneYI2c4x-xGwr-D1X1Ii)

For each language make a copy of the file mentioned above and start translating. Rename the copy by changing [English] to the translated language, e.g. [Mandarin].

Please tell me what you think and report your start and progress as comments on #251

**Comment by niwaa** [comment]
Ok with that. I am ready to start the French translation.

**Comment by lapin7** [comment]
@niwaa OK go ahead and make a copy of the [English] version and name it [French]
The rest of the languages may do the same.

**Comment by noordaa** [comment]
Would like to contribute for the  Serbo-Croat translation of the English .srt file

**Comment by noordaa** [comment]
Professional translators at NATO in Brussels get paid a 0,20 Euro cents per word translated. A word count of this transcript comes to: approx. (leaving numbers and non-text stuff out) 25.000 words. So this text is 5000 Euro investment into Serbian_Croat (and the same could count for the other languages)

**Comment by kaka56** [comment]
Great  am starting  with  Spanish .

**Comment by lapin7** [comment]
Hi @kaka56 I've created [this doc ](https://docs.google.com/document/d/16It5_3fghkhsvGWNVX9MswxPnMlvXCQ0w6aiKotrCxU/edit)for the Spanish version in folder:
 [/RChain/Members/Marketing-Community Dev/YouTube](https://drive.google.com/drive/folders/1zBZXvPMIC9yBneYI2c4x-xGwr-D1X1Ii)

Please work on this doc to show your progress.

**Comment by niwaa** [comment]
Ok i've started the French translation in [this doc](https://docs.google.com/document/d/1en8ewGh88ItE-1UKMYA0wZwz5Elwch0ESfWwD_AiYoo/edit).

**Comment by Tonyprisca13** [comment]
nice am working on Dutch

**Comment by ICA3DaR5** [comment]
I am doing Greek #265 with @thiefinshadows and @zero-andreou

**Comment by BelovedAquila** [comment]
The issue is progressive, nice.was at the verge of getting a reply from from Brad Laurie (Blockchain Brad) only to come see it already resolved. Impressive! 
@lapin7 I want to work on the German translation of the doc, how about that?

**Comment by lapin7** [comment]
@niwaa please make a new issue for the French version. Take #262 as an example.

**Comment by truffard** [comment]
@lapin7 hi  im back.please let me know how i can help with the french translation.

**Comment by lapin7** [comment]
@truffard 
Great that you're back
The translation in French is already busy. Done by @niwaa. 
He's at around 200
@truffard can verify the work done and do what's not done. Please coordinate yourselves.
Tomorrow #272 will be added to 201802: Budgets and Rewards
Progress can be followed in #272

**Comment by lapin7** [comment]
@zsluedem wrote a Python script to count the words. It is about `10954` words in the English transcript regardless the figures or other things.

[The first draft Mandarin transcript](https://docs.google.com/document/d/1nxkJw2Pu1YdeAdWCJz11pVrRKibqnk3NoIZRSO7KxvQ/edit) is done! We are going to review each other's translation now~!

@zsluedem Great.

That brings the budget per language **down**. The previous word count was done by @noordaa, by using the word count function of Google Docs, i.e. 25.000 words. Anyway we have now a good official indicator for translations.

So the budget indication per language becomes **10954 x $0.20 = $2190.8**

**Comment by lapin7** [comment]
@Mervyn853 please contact BlockchainBrad about adding the .srt Chinese version to his video. And the other languages can be added when they're done.

**Comment by lapin7** [comment]
We need also a trailer of this video, max 2 minutes, with the most exiting moments. Anybody can pick this task and create an issue.

**Comment by LiliNL** [comment]
@ HJ I can make a short trailer if you give the timecodes of the moments
that appeal to you the most ...


2018-01-31 17:23 GMT+01:00 HJ Hilbolling <notifications@github.com>:

**Comment by zsluedem** [comment]
@lapin7 @Mervyn853 Chinese transcripts are still need to be reviewed~! 
Don't upload the transcript now until we said we are done~!

**Comment by lapin7** [comment]
@LiliNL please contact @Viraculous, because he said he would make a written summary. Probably he has references to the video moments. You may also contact @entropee, because he is a specialist in making automatic transcripts of a youtube video.
@LiliNL if you go ahead with the trailer, then please make a new issue with a SMART description of what you plan to do. 

PS. if you use the github names then people get notified by mail. So don't use @HJ but @lapin7 ;-)

**Comment by Mervyn853** [comment]
@zsluedem @lapin7 will wait till the translations are finalized before i speak to him again.

**Comment by BelovedAquila** [comment]
@dckc please if you don't mind, what's the China label on almost all issues all about?

**Comment by BelovedAquila** [comment]
@lapin7 To add to the other languages column, I think Portuguese should be one of the languages listed, it's also a key language and I would be working it's transcript.

**Comment by dckc** [comment]
@BelovedAquila the explanation is in https://github.com/rchain/Members/issues/105#issuecomment-362439533 :
> I suggest that a translation into Chinese be labelled **Translation** and **China**. An event in China would be labelled **Events** and **China**. That way someone who notices a Chinese translation might easily find their way to an event in China, and vice versa.

I put the China label only on issues directly relevant to that language / market. There are a lot of them, indeed!

**Comment by BelovedAquila** [comment]
@dckc please permit me to quote you...
"I put the China label only on issues directly relevant to that language / market"
But, using this issue as a reference to your statement above, does this issue seem to be "directly relevant" to just Chinese? Because personally I don't think so.

**Comment by lapin7** [comment]
@Tonyprisca13 please create a new issue for the Dutch version with #272 as a template

**Comment by dckc** [comment]
@BelovedAquila agreed. I labelling this issue with China was a mistake. Fixed.

**Comment by lapin7** [comment]
Please take a look at [this version](https://docs.google.com/document/d/1en8ewGh88ItE-1UKMYA0wZwz5Elwch0ESfWwD_AiYoo/edit#)

From Budget 201802 a range of the issue is copied into the document
From Rewards 201802 a range of the issue is copied into the document
That range can be updated and gives also a link to "Pub ...."

**Comment by zsluedem** [comment]
@Whisker17  and I have finish translating the transcript to Mandarin and we have reviewed each others' work already.
@lapin7 @Mervyn853 feel free to upload the transcript.

**Comment by kaka56** [comment]
@lapin7 i think another key language that should be listed in the translation should be Russian ,i have a close friend that can assist me while i work on the transcript.

**Comment by lapin7** [comment]
@kaka56 If you think it's useful for RChain, go ahead. Don't ask me. ;-)

**Comment by lapin7** [comment]
@Mervyn853 could you take care of all the uploads, because you know BlockchainBrad

**Comment by Mervyn853** [comment]
Sure @lapin7 which are the ones that are completed and ready for upload?

**Comment by lapin7** [comment]
@Mervyn853 Please find the completed ones your self. ;-) 
It's up to you :-)

**Comment by ICA3DaR5** [comment]
The English transcript has many mistakes IMO. I am correcting stuff right now. Will post once I finish.

commas. full-stop's, grammar mistakes are being fixed while watching the video. I am doing this to help with my translation to Greek #265 and the rest of the translations.

**Comment by BelovedAquila** [comment]
@ICA3DaR5 you are right, saw lots of errors while working on a language transcript on it, just had to make the corrections on the transcript am working on. The English transcript used wasn't well reviewed, would also see to reviewing it depending. Good you are working on it.

**Comment by ICA3DaR5** [comment]
The link to the English transcript is wrong: https://docs.google.com/document/d/1oTZVkYEhhnZtyU4DmNiiT3N39R0IVpHI92vP8kaltFs/edit

Takes me to the German file.

**Comment by ICA3DaR5** [comment]
Created the issue #292 working on it for the past 8 hours. I'll probably finish it in 8 hours. :)

**Comment by ICA3DaR5** [comment]
I'll finish #292 tomorrow. Unfortunately I and one of the collaborators were very busy the past two days.

**Comment by ICA3DaR5** [comment]
The English transcript is finished as far as we're concerned. Check #292

**Comment by Viraculous** [comment]
@dckc I think China is a geographical territory, issues tagged with it tends to have a more of a territorial permission than lingual. Chinese should be more appropriate as they are countries not China but speaks Chinese.

**Comment by niwaa** [comment]
Please do make sure you generate a valid .srt file at the end for each language. Bloc #1 until #1625, saved as .srt file, with 1 line separation between each block.  Here is an example: 

-----------

FINAL .srt file for FRENCH subtitles, ready for upload: 
[French_subtitle_youtube_chat_Lawrence_Lerner_Nash_Foster.srt](https://drive.google.com/open?id=1um7HwGa06JexK5MrQ7V-gHVq_A0qEcCV)

**Comment by ICA3DaR5** [comment]
#292 is closed and finished.

**Comment by ICA3DaR5** [comment]
The Greek translation (#265) is finished.

Google Document: https://drive.google.com/open?id=1afa9SDrfS_ZyXpQ9oqLOZWZnzIMmaoZU7w8x2yB-1KA
SRT File: https://drive.google.com/open?id=1bXPyg7U0L7ik8HMIzlHn6RCc5Ipm0Wvy

**Comment by Mervyn853** [comment]
> I have attached the English, Mandarin, French and Greek translations in SRT file formats and would like to request your assistance in uploading the files in your Video Manager section of Youtube. This will allow viewers from all across the world to understand the video. 

Details on the process can be found here: https://support.google.com/youtube/answer/2734796?hl=en

Alternatively, if you're comfortable with it, we could make things easier for you if you could open this specific video to community contributions in which members of the RChain community can contribute to this video with subtitles/cc. 

Details on this process can be found here : https://support.google.com/youtube/answer/6052538

Once you've turned on community contributions, make sure to let your viewers know that they can help with reviewing and translating videos.

Cheers, 
Mervyn

I have sent him the completed SRT files of English, Mandarin, Greek and French and have requested he give community contributor rights so that it will be easier for us to submit directly to the video for him to review. I have also copied you into the email @lapin7

**Comment by kaka56** [comment]
The Spanish  translation [#263](https://github.com/rchain/Members/issues/263) is now complete

**Comment by tcezi** [comment]
which link is to be added to email footer

**Comment by ICA3DaR5** [comment]
@kaka56 If #263 is finished, should I mark it as closed? Also change DRAFT to FINAL here: https://docs.google.com/document/d/1ONyxaJTOL_ZXaFYQWVJ5dEUrnbTm-rCNW10f39sAIXY/edit

**Comment by kaka56** [comment]
@ICA3DaR5  yes we are through with the Spanish translation and changed  the draft  to Final.

**Comment by Chris-Arv** [comment]
Finished with the Italian translation:

Here are the files: Issue #392 

Google Document: https://drive.google.com/open?id=1qx6eMaE6LHotPrnbtsgsS-_h-6FEh0IW3sv-bx3Cuzs
SRT File: https://drive.google.com/open?id=1pBrN6ESjPatZmG0hGGicue8GcOcaZUV9

**Comment by Tonyprisca13** [comment]
@tcezi it should be the link for the video.

**Comment by lapin7** [comment]
@Mervyn853 I don't see any translations on the original video of Brad.
[Exclusive chat Lawrence Lerner & Nash Foster with BlockchainBrad](https://www.youtube.com/watch?v=PwuV7ryhaXw)
How is that possible? Is Brad not cooperating?

**Comment by ICA3DaR5** [comment]
@lapin7 @Mervyn853 doen't answer on Discord either. I have an English and a Greek translation for the RChain Vision as well #363

**Comment by Mervyn853** [comment]
@lapin7 I have included you in the email thread with him. He was nursing his sick daughter for the past week, will try to nudge him again.

Appreciate your kind understanding with this.

**Comment by ICA3DaR5** [comment]
i think #285 is done.

**Comment by flowpoint** [comment]
german should be ready to go #266
fully retranslated retimed version by me: https://drive.google.com/file/d/1lcNd8tK9DWJtLybwpcLehi8gr4p2qZcs/view?usp=sharing
sha256: 80c476af6ccee19dc5479c5dfcbe1672e1a0adf1c36a4dd04c2cd80b1b31b03c

**Comment by ICA3DaR5** [comment]
What's the status of this issue? I see that some money have been payed. Should this be closed since it's overdue?

@lapin7

**Comment by zsluedem** [comment]
@ICA3DaR5  I believed most of the translations have already finished. You can find most of the translation in the [google drive](https://drive.google.com/drive/u/0/folders/1zBZXvPMIC9yBneYI2c4x-xGwr-D1X1Ii). The final step here is to ask [BlockchainBrad](https://www.youtube.com/channel/UCbkjUYiPN8P48r0lurEBP8w) to put up those translations in the Youtube.
@Mervyn853 is the one who connect with `BlockchainBrad`.

For me, it is overdue.
